rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/object/ValidatorsGenerator.xtend

```diff
@@ -149,11 +151,12 @@ class ValidatorsGenerator {
 		if (attr.inf === 0 && attr.isUnbound) {
 			null
 		} else {
+	        /* Casting is required to ensure types are output to ensure code generation in Rosetta */
```

Comment:
This is confusing. Should be "to ensure _recompilation_ in Rosetta", I think.

Reply:
You're right, we can update the comment